sky/templates/gcp-ray.yml.j2

```diff
@@ -28,9 +28,10 @@ provider:
   cache_stopped_nodes: True
   # The GCP project ID.
   project_id: {{gcp_project_id}}
-  # The firewall rule name for customized firewall rules.
-{% if firewall_rule is not none %}
-  firewall_rule: {{firewall_rule}}
+  # The firewall rule name for customized firewall rules. Only enabled
+  # if we have ports requirement.
+{% if have_ports %}
+  firewall_rule: sky-ports-{{cluster_name_on_cloud}}
 {% endif %}
```

Comment:
Ditto. Why not use `GCP.make_deploy_variables` to generate the firewall_rule name?